deployments/aws/templates/all-in-one/all-in-one-with-nlb.yaml

```diff
@@ -751,7 +857,19 @@ Resources:
   SessionManagerPolicies:
     Type: AWS::IAM::Policy
     Properties:
-      PolicyName: !Join ['-', [ai-unlimited, session, !Select [4, !Split ['-', !Select [2, !Split [/, !Ref 'AWS::StackId']]]]]]
+      PolicyName: !Join
+        - '-'
+        - - ai-unlimited
+          - session
+          - !Select
+            - 4
+            - !Split
+              - '-'
+              - !Select
+                - 2
+                - !Split
+                  - /
+                  - !Ref AWS::StackId
       PolicyDocument:
```

Comment:
Please ensure that ec2:DescribeVpcAttribute and ec2:DescribeAvailabilityZones are included on PolicyDocument Statements for ai-unlimited creation.